.eslintrc

```diff
@@ -19,6 +19,8 @@
     "jsx-a11y/no-static-element-interactions": ["off"],
     "react/forbid-prop-types": ["error", { "forbid": ["any"] }],
     "react/jsx-filename-extension": ["error", { "extensions": [".js"] }],
-    "react/require-default-props": ["off"]
+    "react/require-default-props": ["off"],
+    "global-require": ["off"],
+    "no-console": ["warn"]
```

Comment:
Instead of switching off rules completely, just turn them off in that one specific file that you break the rule in. I see only one case in one file for each rule. That doesn't justify disabling them completely.